spec/models/attendance_record_spec.rb

```diff
@@ -17,6 +17,27 @@
     end
   end
 
+  describe 'adds station to student record if entered' do
+    it 'adds station if entered on pair signin' do
+      first_user = FactoryGirl.create(:student)
```

Comment:
`first_user`, `second_user`, `first_attendance_record`, `second_attendance_record` can be DRYed up by adding `let` blocks before the tests are run (see other examples on this page):

```
let(:first_user) { FactoryGirl.create(:student) }
```

Then they can be removed from each individual test. If a record is not explicitly called in a test, but loaded using `let`, use `let!`. This will automatically load the record instead of lazy loading it.